### PR TITLE
Grid Tile Styling for previous entries 

### PR DESCRIPTION
**Pull request by mariah-crawford**
Uses CSS to style the previous entries of the journal to look like mini tiles with pages from a journal

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -15,4 +15,47 @@
...
+    grid-gap: 2px;
+    grid-template-columns: 2.5fr 2fr 0.5fr;
+    width: auto;
+    height: auto;
+}
+.entry .entry-text{
+    padding: 10px;
+    height: 50%;
+    background-position: center center;
+    background-image: url("images/linedpaper.jpg");
```
Comment (gracieorozco) [inline src/main/webapp/style.css:57]: When I ran this branch, there was an error saying that this file was not found. If you can upload the image file, I believe it should resolve the problem.

**Review by gracieorozco — CHANGES_REQUESTED** [review]
Aside from the minor file error, I tested everything else, and it looks great. Once the file is uploaded, I'll verify the branch works once more and approve the changes.

File: src/main/webapp/style.css (hunk 1 above)
Comment (mariah-crawford) [reply src/main/webapp/style.css:57]: Thanks for reviewing my pull request! :) I added the image file and committed the change

**Review by gracieorozco — APPROVED** [review]
It works now, thank you for making the changes. Feel free to merge the branch.

**Pull request merged**